safe/gui/tools/wizard/test/test_keyword_wizard.py

```diff
@@ -1689,7 +1689,11 @@ def test_continuous_raster(self):
             'license': source_license,
             'scale': source_scale,
             'source': source,
-            'thresholds': {'dry': [0.0, 1.0], 'wet': [1.0, 999999.0]},
+            'thresholds': {
+                'high': [1.5, 999999.0],
```

Comment:
I know it's a dictionary, but may be it's better to write it ascending or descending :)